library/ssl_misc.h

```diff
@@ -699,6 +699,9 @@ struct mbedtls_ssl_handshake_params
     unsigned char verify_cookie_len;    /*!<  Cli: cookie length for
                                          *    dtls / tls 1.3
                                          *    Srv: flag for sending a cookie */
+    uint16_t hrr_cookie_len;       /*!<  Cli: hrr cookie length for
```

Comment:
We have to be more precise here. This should rather be something like the following:
```C
#if defined(MBEDTLS_SSL_CLI_C) && \
     ( defined(MBEDTLS_SSL_PROTO_DTLS) || defined(MBEDTLS_SSL_PROTO_TLS1_3) )
    unsigned char *cookie;  /*!< HelloVerifyRequest cookie for DTLS
                             *   HelloRetryRequest cookie for TLS 1.3  */
#endif
#if defined(MBEDTLS_SSL_PROTO_DTLS)
    unsigned char verify_cookie_len;  /*!< Cli: HelloVerifyRequest cookie
                                       *        length
                                       *   Srv: flag for sending a cookie */
#endif
#if defined(MBEDTLS_SSL_CLI_C) && defined(MBEDTLS_SSL_PROTO_TLS1_3)
    uint16_t hrr_cookie_len;  /*!< HelloRetryRequest cookie length */
#endif
```